mpf/platforms/fast/fast.py

```diff
@@ -793,6 +793,10 @@ def get_coil_config_section(cls):
         """Return coil config section."""
         return "fast_coils"
 
+    def validate_coil_section(self, driver, config):
```

Comment:
That does nothing on purpose? There should be a default implementation which actually validates this. Not sure if this is correct. I can check later.

Reply:
Ah yes, I see that now. I'm thinking that these changes may be my own workarounds for overlay platforms that you have since more-correctly fixed. I'll withdraw them from this PR and re-evaluate, thanks!